aws/config

```diff
@@ -0,0 +1,17 @@
+[plugins]
+endpoint = awscli_plugin_endpoint
+
+[default]
+region = ${BUCKET_REGION}
+output = json
+
+s3 =
+  endpoint_url = https://s3.${BUCKET_REGION}.scw.cloud
+  signature_version = s3v4
+  max_concurrent_requests = 100
+  max_queue_size = 1000
+  multipart_threshold = 50 MB
+  multipart_chunksize = 10 MB
+
+s3api =
+  endpoint_url = https://s3.${BUCKET_REGION}.scw.cloud
```

Comment:
```suggestion
  endpoint_url = https://${BUCKET_REGION}.svc.edge.scw.cloud
```

Reply:
@matthprost  no this is the right configuration for uploading on the bucket :)